web/index.js

```diff
@@ -11,12 +11,14 @@
  * This code handles initial render of the bot and any re-renders triggered
  * from javascript events.
  */
-const _ = require('lodash');
-const handlebars = require('handlebars');
+
+import App from './components/App.jsx';
+import isEqual from 'lodash/isEqual';
+import isEmpty from 'lodash/isEmpty';
+const { compile } = require('handlebars');
```

Comment:
why is this different syntax from rest

Reply:
Will change

Reply:
Not sure what's the best practices in javascript, but I do like the fact that this { compile }, reduced the need to add `handlebars.` before each call for the function, also liked the readability of `compile(currentTemplate)`, but then again personal preference, not sure what the standards dictates.
` - const selTemplate = handlebars.compile(currentTemplate);
 + const selTemplate = compile(currentTemplate);`

Reply:
yeah i agree i like it better i think alot of times the whole library is brought in if at the time people think they will be using more than one part in which case i think it makes more sesne to just take whole library but for single functions this makes more sense

Reply:
The main incentive behind the change was that the handlebars module is massive, if we import only what we need like this the webpack bundler can leave out what we don't use and make the total js bundle smaller so our load times are better.